mne/preprocessing/tests/test_ica.py

```diff
@@ -608,4 +608,27 @@ def test_fit_params():
     ICA(fit_params=fit_params)  # test no side effects
     assert_equal(fit_params, {})
 
+
+@requires_sklearn
+def test_bad_channels():
```

Comment:
Check eog is in ica.ch_names






On Sat, Jun 4, 2016 at 1:02 PM -0700, "Eric Larson" <notifications@github.com> wrote:












In mne/preprocessing/tests/test_ica.py:
> @@ -608,4 +608,27 @@ def test_fit_params():
>      ICA(fit_params=fit_params)  # test no side effects
>      assert_equal(fit_params, {})
>  
> +
> +@requires_sklearn
> +def test_bad_channels():

You can fix the random_state, see how it's done in other ICA tests. I'm not
sure what exactly you'd want to test, maybe that it's actually including
the EOG channel or something. Maybe the other existing ICA tests will give
you some ideas.



—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.